### ISSUE TITLE
CLI Linux "invalid_client (401): AADSTS7000215: Invalid client secret is provided."

### ISSUE THREAD

**Issue by FaserF** [bitwarden/directory-connector#179]
Issue:
I get an error that an invalid client secret is provided, but the client secret is definitly correct.
But with the same client secret it is working on the windows GUI variant from the connector. So the client secret is definitly correct.

Versions:
Bitwarden Directory Connector: 2.9.8
Server Installed: 1.44.1 
Web Installed: 2.24.2 

Error:
```
bitwarden@debian:/opt/bitwarden$ bwdc login
Could not find dir, "/home/bitwarden/.config/Bitwarden Directory Connector"; creating it instead.
Could not find data file, "/home/bitwarden/.config/Bitwarden Directory Connector/data.json"; creating it instead.
? client_id: organization.myOrgID
? client_secret: mySecret
{"response":{"error":"invalid_client"},"captchaRequired":false,"statusCode":400}
bitwarden@127:/opt/bitwarden$ bwdc login
? client_id: organization.myOrgID
? client_secret: myNewOtherSecretGenerated
{"response":{"error":"invalid_client"},"captchaRequired":false,"statusCode":400}
bitwarden@127:/opt/bitwarden$ nano /home/bitwarden/.config/Bitwarden\ Directory\ Connector/data.json
------> Here i have copied the working data.json from the windows GUI Connector
bitwarden@127:/opt/bitwarden$ bwdc login
You are already logged in as organization.myOrgID
bitwarden@127:/opt/bitwarden$ bwdc test
AADSTS7000215: Invalid client secret is provided.
Trace ID: 71fe6e1a-5412-433a-8d45-d6a49cd8e800
Correlation ID: a779c5a1-9bbc-46c0-89f0-28e8f74b0105
Timestamp: 2021-11-24 09:17:41Z
invalid_client (401): AADSTS7000215: Invalid client secret is provided.
```

Took the Key from here: 
![image](https://user-images.githubusercontent.com/12374134/143210439-f7d2a465-7b73-44eb-bcef-638fdb9e2971.png)

**Comment by eliykat** [comment]
Hi @FaserF, you need to use your personal API Key to log in using the CLI. See the instructions here: https://bitwarden.com/help/article/personal-api-key/. An organization API Key is only used for the public REST api, not the CLI client.

If you need any further assistance configuring the CLI client, please contact our customer support team using our contact page, here: https://bitwarden.com/contact. You can include a link to this issue in the message content. Thanks!

**Comment by FaserF** [comment]
Hmmm, its a bit misunderstandable. The Windows GUI client explicitly told me that I had to use the organisation api and should not use my user api. And the exactly same data.json works with the windows GUI. 

I have now tried changing it to the user api credentials and get a permission denied error. Maybe I have filled out some variables wrongly, but there is no documentation that I could have a look at. 
I am now in Touch with Tony via Email about this issue.
